src/block/map/util.js

```diff
@@ -106,7 +106,7 @@ const loadScriptAsync = apiKey => {
 		}
 		const script = document.createElement( 'script' )
 		script.id = 'stackable-google-map'
-		script.src = `https://maps.googleapis.com/maps/api/js?key=${ apiKey }&libraries=places`
+		script.src = `https://maps.googleapis.com/maps/api/js?key=${ apiKey }&libraries=places&callback=Function.prototype`
```

Comment:
```suggestion
		// Add callback to prevent warnings.
		script.src = `https://maps.googleapis.com/maps/api/js?key=${ apiKey }&libraries=places&callback=Function.prototype`
```